frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -131,9 +140,31 @@ Provides asynchronous access to specific modules exposed by the plugin.
 
 `utils_bar_ts-chunk.js`: webpack chunk corresponding to the exposed `barUtils` module.
 
+## Plugin development
+
+Run Bridge locally and instruct it to proxy e.g. `/api/plugins/example-plugin` requests directly to your
+local plugin asset server (web server hosting the plugin's generated assets):
+
+```sh
+./bin/bridge -plugins example-plugin=http://localhost:9001/
+```
+
+Your plugin should start loading automatically upon Console application startup. Inspect the value of
+`window.SERVER_FLAGS.consolePlugins` to see the list of plugins which Console loads upon its startup.
+
+## Plugin detection and management
+
+- [Console operator](https://github.com/openshift/console-operator) detects available plugins through
+  `ConsolePlugin` resources on the cluster. It also maintains a cluster-wide list of currently enabled
+  plugins via `spec.plugins` field in its `ConfigMap` object (`console-operator-config`).
```

Comment:
According to https://github.com/openshift/console/pull/8076/files#diff-76eb2371c0e3f846dd818cb2510ab8f9c3d8315cd192aabd781e290970f6793bR57 Console operator uses `Console` object kind (name `cluster`) to store the enabled plugins, not `ConfigMap`

Reply:
OK, maybe @spadgett or @jhadvig can clarify here?

(I thought that enabling a plugin manually involved the editing of `ConfigMap` instance named `console-operator-config` which I thought was the place where the list of enabled plugins was maintained.)

Reply:
The console operator config is a separate CRD in the `operator.openshift.io` group as @rawagner mentions. This is where an admin configures the enabled plugins.

The operator does pass the list in a config map to the console backend, but that's not a public API.

See https://github.com/openshift/enhancements/blob/master/enhancements/console/dynamic-plugins.md#delivering-plugins